### ISSUE TITLE
Implement data chopping

### ISSUE THREAD

**Issue by wdzhou** [neutrons/PyVDrive#25]
It depends on issue #24. 

Work will include
1. adding and pass acceptance test for chopping data (removed as it was dealt with in #24);
2. creating event filters by GUI;
3. chopping data from GUI;
4. view chopping result.

where No.4 is a completely new task, but it is an important method to verify the result.

**Comment by wdzhou** [comment]
Bug: push button 'Add Runs' does not work

**Comment by wdzhou** [comment]
Trivial improvement: set the current working directory as default as to load project.

**Comment by wdzhou** [comment]
Methods to develop:
1. import and export time segments to csv file or other file format which is easy to edit.
2.
